compiler/test/__snapshots__/includes.beda767e.0.snapshot

```diff
@@ -8,7 +8,7 @@ imports › import_all_except_multiple
  (import \"_genv\" \"relocBase\" (global $relocBase_0 i32))
  (import \"_genv\" \"moduleRuntimeId\" (global $moduleRuntimeId_0 i32))
  (import \"GRAIN$MODULE$runtime/gc\" \"GRAIN$EXPORT$incRef\" (global $GRAIN$EXPORT$incRef_0 (mut i32)))
- (import \"GRAIN$MODULE$exportStar\" \"GRAIN$EXPORT$z\" (global $z_1138 (mut i32)))
+ (import \"GRAIN$MODULE$../test-libs/provideAll\" \"GRAIN$EXPORT$x\" (global $x_1126 (mut i32)))
```

Comment:
This is interesting, did we change the way it's being included or did the representation just change?

Reply:
We just changed how it's included in the test. The test this snapshot was originally generated with is gone, but I suppose this is similar enough that git thought it was the same.